### PR TITLE
Support manifests with codename prefixed

### PR DESCRIPTION
**Pull request by oholiab**
This allows manifests to have the release codename explicitly input as the first line, and falls back to the heuristic method of deterimining it from package versions as described in #53

### CODE REVIEW

**Comment by mssalvatore** [comment]
Merged in e327b58e9b2f59697b674be062dc67e4ee57e9ba

File: cvescan/constants.py [hunk 1]
```diff
@@ -105,3 +105,12 @@
 NULL_LOGGER_NAME = "cvescan.null"
 STDOUT_LOGGER_NAME = "cvescan.stdout"
 SYSLOG_LOGGER_NAME = "cvescan.syslog"
+
+SUPPORTED_RELEASES = [
+    "trusty",
+    "xenial",
+    "bionic",
+    "eoan",
```
Comment (stevebeattie) [inline cvescan/constants.py:114]: eoan aka Ubuntu 19.10 has reached end of supported status, and so probably shouldn't be included here.

Comment (mssalvatore) [reply cvescan/constants.py:114]: Good catch. I'll push a fix.

Comment (mssalvatore) [reply cvescan/constants.py:114]: Fixed in 40fe0b7bb8496eee76079823c5ed09e9100bd106

**Pull request closed without merging**